public_html/assets/js/scripts.js

```diff
@@ -164,7 +166,8 @@ $(document).ready(function(){
             logoImgHeight = logoImg.height();
             logoGroupWidth = logoGroup.width();
             logoGroupOffset = logoGroup.offset().left;
-            logoGroupOffsetRight = logoGroupOffset + logoGroupWidth + 30;
+            welcomeSectionDesktopWidth = logoGroupOffset + logoGroupWidth + 50;
+            backgroundImageOffset = welcomeSectionDesktopWidth - 200;
```

Comment:
this brings the background to be near the white background, but tucks underneath -150px.